config/rules/password-secret.yaml

```diff
@@ -16,7 +16,7 @@
 Searcharea: body
 rules:
   - Code: 1002
-    Pattern: (?i)"p[a4@][s$][s$]w[o0]rd([#!@]?(123)?)?"
+    Pattern: (?i)"*p[a4@][s$][s$]w[o0]rd([#!@]?(123)?)?"
```

Comment:
is this PR pertaining to issue -> https://github.com/americanexpress/earlybird/issues/33
Then this does not look like the right way to handle it. We would need to create a new rule to address the pattern.  This particular pattern is looking for the term password.